lib/workout.js

```diff
@@ -123,7 +171,10 @@ module.exports = class Workout extends Content {
       topic: this.topic,
       subtopic: this.subtopic,
       description: this.description,
-      game: this.game
+      section: this.section,
+      type: this.type,
+      game: this.game,
+      exercises: this.exercises
```

Comment:
I'd say remove this from the `toJSON()` for now. otherwise, when merged they'll get sent to the content consumer which will result in errors as exercises are not yet in the `Workout` schema

if we need them in some scripts or w/e you can still do something among these lines:
```
const myWorkout = new Workout(...);
...
const myParsedWorkout = {
  ...myWorkout.toJSON(),
  myWorkout.exercises,
}
```